yarn.lock

```diff
@@ -521,6 +521,11 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
```

Comment:
You didn't make any change in the root package.json, are you sure you want to make this change ?

Reply:
the ts-expect-error comes from using a newer version of `yeoman-enviroment` than the type deffinition in `@types/yeoman-environment`

Reply:
also a big change in the yarn.lock.
there where a few updates, and one conflict so I rebuilt the lock file

Reply:
Ok  👍